medicines/web/Makefile

```diff
@@ -16,3 +16,7 @@ set-env: ## Takes your current .env file and replaces the keyvault value with th
 	--vault-name mhra-dev \
 	--name web-env \
 	--file .env
+
+.PHONY: cypress
+cypress: ## Launch Cypress
+	yarn run cypress open --env $$(cat .env 2> /dev/null | tr "\n", "," | sed 's/,$$/ /g')
```

Comment:
one error when running cypress tests...

```
Navigate to Paracetamol via A-Z indexfailed
CypressError: cy.route() must be called with a url. It can be a string or regular expression.
```